src/components/dialogs/input-hooks.js

```diff
@@ -442,51 +476,131 @@ export const useAutocompleteField = ({
         }
     }, [selectedValue]);
 
+    useEffect(() => {
+        if (!values || typeof values === 'function') {
+            setIsLoading(false);
+            return;
+        }
+        if (typeof values?.then === 'function') {
+            setIsLoading(true);
+        }
+        const valuePromise = Promise.resolve(values);
+        valuePromise.then((vals) => {
+            setPresentedOptions(vals);
+            setIsLoading(false);
+            if (values?.length === 0) setExpanded(false);
+        });
+    }, [values]);
+
+    const handleForcedSearch = useCallback(
+        (term) => {
+            if (!onSearchTermChange) return;
+            setIsLoading(true);
+            setExpanded(true);
+            onSearchTermChange(term, true);
+        },
+        [onSearchTermChange]
+    );
+
+    const myOnOpen = useCallback(() => {
+        setExpanded(true);
+
+        if (!onSearchTermChange) return;
+        if (isWorthLoading(userStr, presentedOptions, userStr, 0)) {
+            setIsLoading(true);
+            onSearchTermChange(userStr, false);
+        }
+    }, [presentedOptions, userStr, onSearchTermChange]);
+
+    const handleSearchTermChange = useCallback(
+        (term) => {
+            if (!onSearchTermChange) return;
+
+            const min = minCharsBeforeSearch;
+
+            setUserStr((old) => {
+                if (isWorthLoading(term, values, old, min)) {
+                    setIsLoading(true);
+                    setExpanded(true);
+                    onSearchTermChange(term, false);
+                }
+                return term;
+            });
+        },
+        [values, minCharsBeforeSearch, onSearchTermChange]
+    );
+
     const field = useMemo(() => {
+        const handleKeyDown = (e) => {
+            if (e.ctrlKey && e.code === 'Space') {
```

Comment:
this key combination seems to be inspired from eclipse usage, I don't think it's well known our users
Also, is it a behaviour we need ?

Reply:
It seems used in some MS software for code-completion : is this fashionable this way ? When not editing (a text), it is usually used to help entering short data.
So user can use it or not be aware, it is not dangerous. And as a developper (that tests the software also by using it) I am happy not to have to remember ids for each equipment type in every network situation format and geographical area.

Reply:
I understand it's useful as a developer, but is it a behaviour that was asked for production ? Hidden undocumented behaviour can lead to problems, I'm not sure we should create shortcuts that will go to production to help us test the app